x-pack/plugins/observability/public/pages/overview/empty_section.ts

```diff
@@ -97,7 +97,7 @@ export const getEmptySections = ({ http }: { http: HttpSetup }): ISection[] => {
       linkTitle: i18n.translate('xpack.observability.emptySection.apps.alert.link', {
         defaultMessage: 'Create rule',
       }),
-      href: http.basePath.prepend('/app/management/insightsAndAlerting/triggersActions/alerts'),
```

Comment:
Also, I am wondering why this one goes to `/alerts`. From the ticket, it seems all the places are redirected to `/rules` not `/alerts` 🤔

Reply:
Have updated to `/rules` for now.